dev-haskell/web-encodings/web-encodings-0.3.0.9-r1.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
```

Comment:
`epatch` (used below) is not allowed in `EAPI=7`. `eapply_user` or `default` needs to be called in `src_prepare()` in `EAPI=7`:

```
>>> Preparing source in /tmp/portage/dev-haskell/web-encodings-0.3.0.9-r1/work/web-encodings-0.3.0.9 ...
/tmp/portage/dev-haskell/web-encodings-0.3.0.9-r1/temp/environment: строка 1972: epatch: команда не найдена
 * CHDEP: 'bytestring >= 0.9.1.4 && < 0.10' -> 'bytestring >= 0.9.1.4'
 * CHDEP: 'directory >= 1 && < 1.2' -> 'directory >= 1'
 * CHDEP: 'failure >= 0.0.0 && < 0.2' -> 'failure >= 0.0.0'
 * CHDEP: 'text >= 0.11 && < 0.12' -> 'text >= 0.11'
 * ERROR: dev-haskell/web-encodings-0.3.0.9-r1::haskell failed (prepare phase):
 *   eapply_user (or default) must be called in src_prepare()!
 *
 * Call stack:
 *            ebuild.sh, line  762:  Called __ebuild_main 'prepare'
 *   phase-functions.sh, line 1050:  Called __dyn_prepare
 *   phase-functions.sh, line  395:  Called die
 * The specific snippet of code:
 *              die "eapply_user (or default) must be called in src_prepare()!"
```

Reply:
Very good point, I will re-work this PR to only affect ebuilds WITHOUT any `src_*` or `pkg_*` as you've suggested. I'll create a separate PR altogether for these last stragglers.